packages/bpk-component-page-indicator/src/BpkPageIndicator.js

```diff
@@ -57,91 +57,114 @@ const BpkPageIndicator = ({
   showNav,
   totalIndicators,
   variant,
-}: Props) => (
-  // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'.
-  <div className={className}>
+}: Props) => {
+  const isOverImage = variant === VARIANT.overImage;
+  const isInteractive = !!onClick;
```

Comment:
Could you clarify or add a comment as the purpose of this?

Reply:
For `isOverImage`, what I did this change before I was thinking this component should include the according to `css` when customer pass `VARIANT.overImage ` and in order to be compatible with the custom bottom spacing, customer need pass one more prop `bottom`, it seems too many changes for this existing component, so I planed to move this css for `overImage` to `BpkCarousel`, what do you think about it?

For `isInteractive`, I will add some comments.